package.json

```diff
@@ -1,14 +1,20 @@
 {
   "name": "@codacy/api-typescript",
-  "version": "1.0.0",
+  "version": "0.0.1",
+  "apiVersion": "5.12.0",
   "description": "Codacy API client for Typescript",
   "main": "./dist/codacy-api-client.js",
-  "module": "./esm/client.js",
-  "types": "./esm/client.d.ts",
+  "module": "./lib/client.js",
+  "types": "./lib/client.d.ts",
   "scripts": {
     "build": "tsc && rollup -c rollup.config.js && npm run minify",
```

Comment:
What is the purpose of having all the rollup, and uglifyjs stuff?
Shouldn't those details be a problem of the end user of this library?

Ideally we could just have the esm or es5 version, right?

Reply:
Probably **yes**. I just respected the generated base structure of the autorest generator and worked over it.